Dockerfile

```diff
@@ -14,8 +14,8 @@ RUN set -ex \
         /var/tmp/* \
         /usr/share/doc \
         /usr/share/doc-base \
-    && SLUGIFY_USES_TEXT_UNIDECODE=yes pip install --no-cache-dir "apache-airflow[devel_hadoop,crypto,celery,postgres,jdbc,ssh]==1.10.1" psycopg2 \
-    && pip install redis==2.10.6
```

Comment:
Qn: Why was `redis` previously needed, but now not needed anymore?

Reply:
Redis is now installed as part of airflow python package as indicated the previous line as the redis dependency is now fixed: https://github.com/apache/airflow/pull/4195

See also https://github.com/apache/airflow/blob/cb8b2a1dc64c3ea6ba445893c65c6c953dfb476a/setup.py#L230

Reply:
Okay thanks for clarification :+1: